docs/databases/connections/postgresql.md

```diff
@@ -58,11 +58,19 @@ If you set the SSL Mode to either "verify-ca" or "verify-full", you'll need to s
 
 You have the option of using a **Local file path** or an **Uploaded file path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
 
-#### SSL Client KEY (PKCS-8/DER)
+#### SSL Client Key (PKCS-8/DER)
 
-Again, you have the option of using a **Local file path** or an **Uploaded file path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
+Again, you have the option of using a **Local file path** or an **Uploaded file path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate. You'll also need to input your **SSL Client Key Password**.
 
-You'll also need to input your **SSL Client Key Password**.
+The private key must be PKCS8 and stored in DER format.
+
+If you instead have a PEM SSL client key, you can convert that key to the PKCS-8/DER format using [openssl](https://www.openssl.org/). The command would look something like:
+
+```
+openssl pkcs8 -topk8 -inform PEM -outform DER -in client-key.pem -out client-key.pk8 -nocrypt
+```
+
+Use either `sslmode=require` or `verify-ca`, enter your converted client key in Metabase, and you should be good to go.
```

Comment:
Technically the customer should select `sslmode` depending on what level of security they expect/require. The differences between the different modes are detailed in the postgres docs [here](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS); maybe we should just link to this?

Reply:
@noahmoss. Earlier in the page, we link to the [jdbc docs](https://jdbc.postgresql.org/documentation/ssl/#configuring-the-client) that detail the security levels. Should we just omit this paragraph (line 73)?

Reply:
Yeah I think we can omit it. My main point is that we shouldn't imply that the choice of `sslmode` is arbitrary.